### DISCUSSION TITLE
ReadMe stats

### DISCUSSION THREAD

**Discussion by sijirama** [anuraghazra/github-readme-stats#2516] [Q&A] [answered]
How are the readme stats calculated, what makes some in A or A+

**Comment by mustafaozhan** [comment, marked as answer]
> Note Available ranks are S+ (top 1%), S (top 25%), A++ (top 45%), A+ (top 60%), and B+ (everyone). The values are calculated by using the [cumulative distribution function](https://en.wikipedia.org/wiki/Cumulative_distribution_function) using commits, contributions, issues, stars, pull requests, followers, and owned repositories. The implementation can be investigated at [src/calculateRank.js](https://github.com/anuraghazra/github-readme-stats/blob/master/src/calculateRank.js).

@sijirama this is from readme of the repo

**Reply by sijirama** [reply]
Thanks
